test/e2e/arktos/multi_tenancy/test_suites/multi_tenancy_controller/test_deployment_replicaset_controller.yaml

```diff
@@ -143,6 +143,4 @@ Tests:
 # cleanup
 ######################################################################################################
   - BeforeTestMessage: Clean up...
-    Command: ${kubectl} delete tenant ${test_tenant}
```

Comment:
why these were removed?

Reply:
Gosh, my bad. That is why we need peer review for code change. Thanks for catching this.

Fixed.

Reply:
ah understand. thanks.